bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/handler/ConditionHandler.java

```diff
@@ -29,6 +29,6 @@
      *            defined in form: ModuleId.outputId
      * @return true if Condition is satisfied, false otherwise
      */
-    public boolean isSatisfied(Map<String, ?> context);
+    public boolean isSatisfied(Map<String, Object> context);
```

Comment:
Shouldn't we change the JavaDoc to inform the implementer that the context argument is unmodifiable?